### PR TITLE
API: Enable serving Swagger UI by default and add docs and guidelines

### PR DESCRIPTION
**Pull request by papagian**
**What is this feature?**

* removes `swaggerUi` feature toggle
* enables serving the Swagger UI by default (it used to be behind the feature flag)
* updates HTTP API docs
* adds developer guidelines for maintaining the specifications

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/developers/http_api/_index.md
Comment (joshhunt) [inline docs/sources/developers/http_api/_index.md:26]:
```suggestion
Users can browser and try out both via the Swagger UI editor (served by the grafana server) by navigating to `/swagger-ui` and `/openapi3` respectively.
```

**Review by chri2547 — COMMENTED** [review]
The feature toggles docs removal LGTM, so approved pending Engr approval. Including @josmperez to take a look at the addition to the dev docs. Thank you for the contribution!

File: pkg/api/README.md
Comment (josmperez) [inline pkg/api/README.md:3]: *an* OpenAPI v3 specification

File: pkg/api/README.md
Comment (josmperez) [inline pkg/api/README.md:7]: could link text be more descriptive? re: https://grafana.com/docs/writers-toolkit/writing-guide/useful-link-text/

File: pkg/api/README.md
Comment (josmperez) [inline pkg/api/README.md:13]: I think the Google style for ID is "enter your ID" but "enter the id attribute", so ID would probably be preferred here.

File: pkg/api/README.md
Comment (josmperez) [inline pkg/api/README.md:38]: Propose changing "the following" to "the following route"

File: pkg/api/README.md
Comment (josmperez) [inline pkg/api/README.md:55]: as before, I propose changing "following" into adjective.

**Review by josmperez — CHANGES_REQUESTED** [review]
Minor editorial suggestions.

**Review by mckn — APPROVED** [review]
LGTM!

File: pkg/api/README.md
Comment (papagian) [reply pkg/api/README.md:38]: what about `the following block`? the code does not actually describes a route but rather a struct with the parameters that the route expects.

**Comment by papagian** [comment]
@josmperez I made some changes following your review; can you check once again, thank you!

**Review by josmperez — APPROVED** [review]

**Comment by BartG95** [comment]
Please be advised that this feature loads stuff from unpkg.com, which to me is a third party. Both technical measures such as CSP, and legal measures such as the GDPR, may prevent this from working.

**Pull request merged**
